packages/components/psammead-headings/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "React styled components for a Headline and SubHeading",
   "repository": {
     "type": "git",
-    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/utilities/psammead-headings"
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-headings"
```

Comment:
As this updates the content on the NPM package we should bump and publish this change